rails_example/app/workers/simple_worker.rb

```diff
@@ -3,7 +3,7 @@ class SimpleWorker
   sidekiq_options unique: :until_executed,
                   unique_args: -> (args) { [args.first] },
                   queue: :default,
-                  default_expiration: 3 * 60 * 60
+                  default_queue_lock_expiration: 3 * 60 * 60
```

Comment:
Still thinking about this but maybe `default_queue_expiration` and `default_run_expiration` would suffice. We already know it is expiration of a lock so it feels redundant. On the other hand it is very explicit. What do you think?